core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -323,49 +290,39 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
       }
     }
 
-  def listWorkspaces(): Future[PerRequestMessage] =
-    dataSource.inTransaction ({ dataAccess =>
-
-      val query = for {
-        permissionsPairs <- listWorkspaces(RawlsUser(userInfo), dataAccess)
-        managedGroupsForUser <- DBIO.from(samDAO.listManagedGroups(userInfo))
-        ownerEmails <- dataAccess.workspaceQuery.listAccessGroupMemberEmails(permissionsPairs.map(p => UUID.fromString(p.workspaceId)), WorkspaceAccessLevels.Owner)
-        publicWorkspaces <- dataAccess.workspaceQuery.listWorkspacesWithGroupAccess(permissionsPairs.map(p => UUID.fromString(p.workspaceId)), UserService.allUsersGroupRef)
-        submissionSummaryStats <- dataAccess.workspaceQuery.listSubmissionSummaryStats(permissionsPairs.map(p => UUID.fromString(p.workspaceId)))
-        workspaces <- dataAccess.workspaceQuery.listByIds(permissionsPairs.map(p => UUID.fromString(p.workspaceId)))
-      } yield (permissionsPairs, managedGroupsForUser, ownerEmails, publicWorkspaces, submissionSummaryStats, workspaces)
-
-      val results = query.map { case (permissionsPairs, managedGroupsForUser, ownerEmails, publicWorkspaces, submissionSummaryStats, workspaces) =>
-        val workspacesById = workspaces.groupBy(_.workspaceId).mapValues(_.head)
-        permissionsPairs.map { permissionsPair =>
-          val groupsForUser = managedGroupsForUser.map(group => ManagedGroupRef(group.groupName)).toSet
-
-          workspacesById.get(permissionsPair.workspaceId).map { workspace =>
-            def trueAccessLevel = {
-              if((workspace.authorizationDomain -- groupsForUser).nonEmpty) WorkspaceAccessLevels.NoAccess
-              else permissionsPair.accessLevel
+  def listWorkspaces(lite: Boolean): Future[PerRequestMessage] = {
```

Comment:
context: i put this in here with the intention of making fewer unnecessary calls to sam (for reading the owner policy for each workspace). didn't end up doing anything with it because i didn't have any proof that it'd actually be a noticeable improvement. would be interested to know how listWorkspaces performs with several hundred workspaces. if it's poor, then this would be useful for fc-ui, saturn, etc

Reply:
I am removing lite but I think my plan is to also remove the owners from the list workspace call anyway as it is not displayed.